### ISSUE TITLE
The video's resolution is not supported

### ISSUE THREAD

**Issue by sourpatched** [jellyfin/jellyfin-androidtv#1576]
### Describe the bug

Server running 10.8.0 beta

3 different videos all transcode because of The video's resolution is not supported, they used to direct play on Jellyfin 10.7.0 
2 video files are HD and the other one is SD. ATV plays 99% of my content as direct play/

Video
Title: 720p H264 SDR
Codec: H264
AVC: No
Profile: Main
Level: 31
Resolution: 1280x720
Aspect ratio: 16:9
Interlaced: No
Framerate: 29.97003
Bitrate: 1326 kbps
Bit depth: 8 bit
Video range: SDR
Color space: bt709
Color transfer: bt709
Color primaries: bt709
Pixel format: yuv420p
Ref frames: 1
NAL: 0

Video
Title: 720p H264 SDR
Codec: H264
AVC: No
Profile: High
Level: 31
Resolution: 1280x720
Aspect ratio: 16:9
Interlaced: No
Framerate: 29.97003
Bitrate: 3320 kbps
Bit depth: 8 bit
Video range: SDR
Color space: bt709
Color transfer: bt709
Color primaries: bt709
Pixel format: yuv420p
Ref frames: 1
NAL: 0

Video
Title: 480p H264 SDR
Codec: H264
Codec tag: avc1
AVC: No
Profile: High
Level: 30
Resolution: 720x400
Aspect ratio: 16:9
Interlaced: No
Framerate: 23.976025
Bitrate: 618 kbps
Bit depth: 8 bit
Video range: SDR
Pixel format: yuv420p
Ref frames: 1
NAL: 0

### Logs

```shell
Duration: 00:44:05.81, start: 0.000000, bitrate: 754 kb/s
  Stream #0:0(und): Video: h264 (High) (avc1 / 0x31637661), yuv420p, 720x400 [SAR 1:1 DAR 9:5], 618 kb/s, 23.98 fps, 23.98 tbr, 24k tbn, 47.95 tbc (default)
    Metadata:
      handler_name    : VideoHandler
      vendor_id       : [0][0][0][0]
  Stream #0:1(und): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 130 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
      vendor_id       : [0][0][0][0]
  Stream #0:2: Video: mjpeg (Progressive), yuvj420p(pc, bt470bg/unknown/unknown), 400x578 [SAR 1:1 DAR 200:289], 90k tbr, 90k tbn, 90k tbc (attached pic)
Stream mapping:
  Stream #0:0 -> #0:0 (h264 (native) -> h264 (libx264))
  Stream #0:1 -> #0:1 (aac (native) -> aac (native))
Press [q] to stop, [?] for help
[libx264 @ 0x55f08b8c9bc0] using SAR=1/1
[libx264 @ 0x55f08b8c9bc0] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX FMA3 BMI2 AVX2
[libx264 @ 0x55f08b8c9bc0] profile Main, level 3.0
[libx264 @ 0x55f08b8c9bc0] 264 - core 155 r2917 0a84d98 - H.264/MPEG-4 AVC codec - Copyleft 2003-2018 - http://www.videolan.org/x264.html - options: cabac=1 ref=1 deblock=1:0:0 analyse=0x1:0 me=dia subme=0 psy=1 psy_rd=1.00:0.00 mixed_ref=0 me_range=4 chroma_me=0 trellis=0 8x8dct=0 cqm=0 deadzone=21,11 fast_pskip=1 chroma_qp_offset=0 threads=6 lookahead_threads=1 sliced_threads=0 nr=0 decimate=1 interlaced=0 bluray_compat=0 constrained_intra=0 bframes=3 b_pyramid=2 b_adapt=1 b_bias=0 direct=1 weightb=1 open_gop=0 weightp=1 keyint=250 keyint_min=23 scenecut=40 intra_refresh=0 rc_lookahead=10 rc=crf mbtree=0 crf=23.0 qcomp=0.60 qpmin=0 qpmax=69 qpstep=4 vbv_maxrate=3865 vbv_bufsize=7730 crf_max=0.0 nal_hrd=none filler=0 ip_ratio=1.40 pb_ratio=1.30 aq=1:1.00
Output #0, matroska, to '/var/lib/jellyfin/transcodes/6cc883685841bdabdf1a008baf89f9fe.mkv':
  Metadata:
    encoder         : Lavf58.76.100
  Stream #0:0: Video: h264 (H264 / 0x34363248), yuv420p(bt709, progressive), 720x400 [SAR 1:1 DAR 9:5], q=2-31, 23.98 fps, 1k tbn (default)
    Metadata:
      encoder         : Lavc58.134.100 libx264
    Side data:
      cpb: bitrate max/min/avg: 3865000/0/0 buffer size: 7730000 vbv_delay: N/A
  Stream #0:1: Audio: aac (LC) ([255][0][0][0] / 0x00FF), 48000 Hz, stereo, fltp, 130 kb/s (default)

[{"Codec":"h264","CodecTag":"avc1","Language":"und","ColorRange":null,"ColorSpace":null,"ColorTransfer":null,"ColorPrimaries":null,"Comment":null,"TimeBase":"1/24000","CodecTimeBase":null,"Title":null,"VideoRange":"SDR","LocalizedUndefined":null,"LocalizedDefault":null,"LocalizedForced":null,"LocalizedExternal":null,"DisplayTitle":"480p H264 SDR","NalLengthSize":"0","IsInterlaced":false,"IsAVC":false,"ChannelLayout":null,"BitRate":618410,"BitDepth":8,"RefFrames":1,"PacketLength":null,"Channels":null,"SampleRate":null,"IsDefault":true,"IsForced":false,"Height":400,"Width":720,"AverageFrameRate":23.976025,"RealFrameRate":23.976025,"Profile":"High","Type":1,"AspectRatio":"16:9","Index":0,"Score":null,"IsExternal":false,"DeliveryMethod":null,"DeliveryUrl":null,"IsExternalUrl":null,"IsTextSubtitleStream":false,"SupportsExternalStream":false,"Path":null,"PixelFormat":"yuv420p","Level":30,"IsAnamorphic":null},{"Codec":"aac","CodecTag":"mp4a","Language":"und","ColorRange":null,"ColorSpace":null,"ColorTransfer":null,"ColorPrimaries":null,"Comment":null,"TimeBase":"1/48000","CodecTimeBase":null,"Title":null,"VideoRange":null,"LocalizedUndefined":null,"LocalizedDefault":null,"LocalizedForced":null,"LocalizedExternal":null,"DisplayTitle":"AAC - Stereo - Default","NalLengthSize":null,"IsInterlaced":false,"IsAVC":null,"ChannelLayout":"stereo","BitRate":130823,"BitDepth":null,"RefFrames":null,"PacketLength":null,"Channels":2,"SampleRate":48000,"IsDefault":true,"IsForced":false,"Height":null,"Width":null,"AverageFrameRate":null,"RealFrameRate":null,"Profile":"LC","Type":0,"AspectRatio":null,"Index":1,"Score":null,"IsExternal":false,"DeliveryMethod":null,"DeliveryUrl":null,"IsExternalUrl":null,"IsTextSubtitleStream":false,"SupportsExternalStream":false,"Path":null,"PixelFormat":null,"Level":0,"IsAnamorphic":null},{"Codec":"mjpeg","CodecTag":null,"Language":null,"ColorRange":null,"ColorSpace":"bt470bg","ColorTransfer":null,"ColorPrimaries":null,"Comment":null,"TimeBase":"1/90000","CodecTimeBase":null,"Title":null,"VideoRange":null,"LocalizedUndefined":null,"LocalizedDefault":null,"LocalizedForced":null,"LocalizedExternal":null,"DisplayTitle":null,"NalLengthSize":null,"IsInterlaced":false,"IsAVC":null,"ChannelLayout":null,"BitRate":null,"BitDepth":8,"RefFrames":1,"PacketLength":null,"Channels":null,"SampleRate":null,"IsDefault":false,"IsForced":false,"Height":578,"Width":400,"AverageFrameRate":null,"RealFrameRate":90000,"Profile":"Progressive","Type":3,"AspectRatio":"200:289","Index":2,"Score":null,"IsExternal":false,"DeliveryMethod":null,"DeliveryUrl":null,"IsExternalUrl":null,"IsTextSubtitleStream":false,"SupportsExternalStream":false,"Path":null,"PixelFormat":"yuvj420p","Level":-99,"IsAnamorphic":null}],"MediaAttachments":[],"Formats":[],"Bitrate":754796,"Timestamp":null,"RequiredHttpHeaders":{},"TranscodingUrl":null,"TranscodingSubProtocol":null,"TranscodingContainer":null,"AnalyzeDurationMs":null,"DefaultAudioStreamIndex":null,"DefaultSubtitleStreamIndex":null}
```

### Application version

0.13.2

### Where did you install the app from?

Sideloaded APK

### Device information

SHIELD pro

### Android version

Android 11

### Jellyfin server version

10.8.0 beta

**Comment by bentrop** [comment]
I believe I'm seeing the exact same issue on my setup (Sony Bravia with AndroidTV 9.0, AndroidTV client 0.13.2 downloaded via Play Store).

LOG:
`[2022-03-30 20:47:28.543 +02:00] [INF] [17] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for "AndroidTV". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: False EnableAudioPlaybackTranscoding: False
[2022-03-30 20:47:28.552 +02:00] [INF] [17] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile="AndroidTV-libVLC", Path="F:\Videos\TV Shows\Bluey\Bluey.mp4", AudioStreamIndex=null, SubtitleStreamIndex=null ) => ( PlayMethod=DirectPlay, TranscodeReason=0 ) "media:/videos/5c5e074c-dca1-3540-fafb-9ac19968bbdf/stream.mov?MediaSourceId=5c5e074cdca13540fafb9ac19968bbdf&Static=true&VideoCodec=h264&AudioCodec=aac&AudioStreamIndex=1&api_key=<token>&SubtitleMethod=Encode&Tag=08c90265130ada725273d151b4004c0f"
[2022-03-30 20:47:28.833 +02:00] [INF] [17] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for "AndroidTV". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: False EnableAudioPlaybackTranscoding: False
[2022-03-30 20:47:28.840 +02:00] [INF] [17] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile="AndroidTV-ExoPlayer", Path="F:\Videos\TV Shows\Bluey\Bluey.mp4", AudioStreamIndex=null, SubtitleStreamIndex=null ) => ( PlayMethod=Transcode, TranscodeReason=VideoResolutionNotSupported ) "media:/videos/5c5e074c-dca1-3540-fafb-9ac19968bbdf/stream?MediaSourceId=5c5e074cdca13540fafb9ac19968bbdf&AudioStreamIndex=1&api_key=<token>&SubtitleMethod=Encode&RequireAvc=false&Tag=08c90265130ada725273d151b4004c0f&TranscodeReasons=VideoResolutionNotSupported"`

The file direct played just fine on the AndroidTV 0.13 client with Jellyfin Server 10.7. Upgrading to 10.8-beta1 broke it. Transcoding is disabled on the server.

The file still plays fine on all alternative Jellyfin clients: MrMC and Kodi+Plugin - it's only broken on the latest AndroidTV client with the current 10.8-beta server.

**Comment by sourpatched** [comment]
An Update: This should be fixed in 10.8.0 But AndroidTV profile may not be perfect.

**Comment by sourpatched** [comment]
Closing as I believe that this is no longer an issue :)
